gulpfile.js

```diff
@@ -5,6 +5,7 @@ var del = require('del');
 var rename = require("gulp-rename");
 var jshint = require('gulp-jshint');
 var stylish = require('jshint-stylish');
+const eslint = require('gulp-eslint');
```

Comment:
Note: Other `require`s could be made constants .

Reply:
Done